docs/source/examples.select_fps.ipynb

```diff
@@ -0,0 +1,102 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "178bd73f-5197-4eb5-8a1e-f1bf16fad4cc",
+   "metadata": {},
+   "source": [
+    "# Selection or Sampling of Structures\n",
+    "\n",
+    "Training of machine learning potentials often requires picking-out a few structures from a large configurations-database (for example, an MD trajectory). This can be achieved in a few different ways in the Workflow package depending on one's choice of the selection criteria (see `wfl.select`).\n",
+    "\n",
+    "Selection of a set of individually unique structures can be done by comparing descriptors for each configuration in the database (see `wfl.select.by_descriptor`). Here, you can find functions to process descriptors as well as functions which perform two different selection algorithms, namely leverage-score CUR and greedy farthest-point-first (FPS). Additional features including exclusion of a list of structures or consideration of previously selected structures can be passed as arguments in the selection criteria.\n",
+    "\n",
+    "In this example, we show how FPS can be used for selection of \"n=8\" datapoints from an MD trajectory by comparing \"average SOAP descriptors\" of all path configurations. This is done in two steps: \n",
+    "1. Assigning  a global descriptor for each configuration in the trajectory followed by \n",
+    "2. A call of the *greedy_fps_conf_global* function \n",
+    "\n",
+    "To assign a per-config descriptor we calculate the average SOAP vector for every frame in the MD trajectory. The greedy-FPS algorithm would use them to measure similarities across the datapoints and select 10 unique structures. \n",
+    "\n",
+    "Overall this requires two input files: the database and the descriptors (\"md.traj\" and \"params.yaml\")\n",
+    "\n",
+    "Tip: params.yaml can either be self-written or automatically generated from a \"Univeral_SOAP-template\" processed by multi-stage gap fit (see wfl.fit.gap.multistage)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a209cdf-1153-4e7d-bedb-dcd8cea04116",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wfl, os, sys, yaml\n",
+    "from wfl.configset import ConfigSet, OutputSpec\n",
+    "from wfl.descriptors.quippy import calc as calc_descriptors\n",
+    "from wfl.select.by_descriptor import greedy_fps_conf_global as get_fps\n",
+    "\n",
+    "work_dir = os.path.join(os.path.dirname(wfl.__file__), \"../docs/source/examples_files/select_fps\") # Location of I/O files\n",
```

Comment:
I generally prefer `pathlib.Path` over `os` for file handling, but that's not a requirement.